swarmrl/agents/actor_critic.py

```diff
@@ -172,7 +172,7 @@ def calc_action(self, colloids: typing.List[Colloid]) -> typing.List[Action]:
         action_indices, log_probs = self.network.compute_action(
             observables=state_description
         )
-        chosen_actions = np.take(list(self.actions.values()), action_indices, axis=-1)
+        chosen_actions = np.take(list(self.actions.values()), action_indices, axis=0)#for Resobee "0", for Espresso "-1"
```

Comment:
This is where, for example, if you moved the resobee actions were classes, the code would be compatible.

Reply:
ResoBee now also uses the action class.